app/assets/stylesheets/custom.scss

```diff
@@ -1,3 +1,8 @@
+body {
+  font-family: 'Ubuntu', sans-serif;
+  font-weight: bold;
+}
+
 .home-search {
   background:rgba(0 ,0 ,0 ,0.5);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space